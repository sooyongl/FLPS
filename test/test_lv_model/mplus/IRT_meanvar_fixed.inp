data: file is irt.csv;
variable: names are x1-x10;
categorical = x1-x10;

analysis: estimator = ml;
	  
model:

F by x1* x2-x10;

[F@0];
F@1;