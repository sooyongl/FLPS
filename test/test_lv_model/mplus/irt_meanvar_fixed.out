Mplus VERSION 8.6
MUTHEN & MUTHEN
11/18/2021   5:20 PM

INPUT INSTRUCTIONS

  data: file is irt.csv;
  variable: names are x1-x10;
  categorical = x1-x10;

  analysis: estimator = ml;
  	
  model:

  F by x1* x2-x10;

  [F@0];
  F@1;



INPUT READING TERMINATED NORMALLY




SUMMARY OF ANALYSIS

Number of groups                                                 1
Number of observations                                         500

Number of dependent variables                                   10
Number of independent variables                                  0
Number of continuous latent variables                            1

Observed dependent variables

  Binary and ordered categorical (ordinal)
   X1          X2          X3          X4          X5          X6
   X7          X8          X9          X10

Continuous latent variables
   F


Estimator                                                       ML
Information matrix                                        OBSERVED
Optimization Specifications for the Quasi-Newton Algorithm for
Continuous Outcomes
  Maximum number of iterations                                 100
  Convergence criterion                                  0.100D-05
Optimization Specifications for the EM Algorithm
  Maximum number of iterations                                 500
  Convergence criteria
    Loglikelihood change                                 0.100D-02
    Relative loglikelihood change                        0.100D-05
    Derivative                                           0.100D-02
Optimization Specifications for the M step of the EM Algorithm for
Categorical Latent variables
  Number of M step iterations                                    1
  M step convergence criterion                           0.100D-02
  Basis for M step termination                           ITERATION
Optimization Specifications for the M step of the EM Algorithm for
Censored, Binary or Ordered Categorical (Ordinal), Unordered
Categorical (Nominal) and Count Outcomes
  Number of M step iterations                                    1
  M step convergence criterion                           0.100D-02
  Basis for M step termination                           ITERATION
  Maximum value for logit thresholds                            15
  Minimum value for logit thresholds                           -15
  Minimum expected cell size for chi-square              0.100D-01
Optimization algorithm                                         EMA
Integration Specifications
  Type                                                    STANDARD
  Number of integration points                                  15
  Dimensions of numerical integration                            1
  Adaptive quadrature                                           ON
Link                                                         LOGIT
Cholesky                                                        ON

Input data file(s)
  irt.csv
Input data format  FREE


UNIVARIATE PROPORTIONS AND COUNTS FOR CATEGORICAL VARIABLES

    X1
      Category 1    0.630          315.000
      Category 2    0.370          185.000
    X2
      Category 1    0.616          308.000
      Category 2    0.384          192.000
    X3
      Category 1    0.486          243.000
      Category 2    0.514          257.000
    X4
      Category 1    0.434          217.000
      Category 2    0.566          283.000
    X5
      Category 1    0.468          234.000
      Category 2    0.532          266.000
    X6
      Category 1    0.520          260.000
      Category 2    0.480          240.000
    X7
      Category 1    0.622          311.000
      Category 2    0.378          189.000
    X8
      Category 1    0.502          251.000
      Category 2    0.498          249.000
    X9
      Category 1    0.256          128.000
      Category 2    0.744          372.000
    X10
      Category 1    0.450          225.000
      Category 2    0.550          275.000



THE MODEL ESTIMATION TERMINATED NORMALLY



MODEL FIT INFORMATION

Number of Free Parameters                       20

Loglikelihood

          H0 Value                       -3184.955

Information Criteria

          Akaike (AIC)                    6409.911
          Bayesian (BIC)                  6494.203
          Sample-Size Adjusted BIC        6430.722
            (n* = (n + 2) / 24)

Chi-Square Test of Model Fit for the Binary and Ordered Categorical
(Ordinal) Outcomes

          Pearson Chi-Square

          Value                            946.211
          Degrees of Freedom                  1003
          P-Value                           0.8993

          Likelihood Ratio Chi-Square

          Value                            768.051
          Degrees of Freedom                  1003
          P-Value                           1.0000



MODEL RESULTS

                                                    Two-Tailed
                    Estimate       S.E.  Est./S.E.    P-Value

 F        BY
    X1                 0.912      0.158      5.779      0.000
    X2                 1.509      0.236      6.400      0.000
    X3                 0.868      0.153      5.689      0.000
    X4                 0.699      0.139      5.029      0.000
    X5                 0.401      0.120      3.346      0.001
    X6                 1.479      0.227      6.510      0.000
    X7                 1.281      0.200      6.415      0.000
    X8                 0.395      0.120      3.308      0.001
    X9                 1.249      0.215      5.823      0.000
    X10                0.866      0.152      5.702      0.000

 Means
    F                  0.000      0.000    999.000    999.000

 Thresholds
    X1$1               0.623      0.112      5.583      0.000
    X2$1               0.661      0.137      4.824      0.000
    X3$1              -0.067      0.104     -0.646      0.518
    X4$1              -0.296      0.101     -2.940      0.003
    X5$1              -0.133      0.093     -1.433      0.152
    X6$1               0.105      0.126      0.838      0.402
    X7$1               0.650      0.126      5.143      0.000
    X8$1               0.008      0.093      0.086      0.931
    X9$1              -1.378      0.156     -8.825      0.000
    X10$1             -0.235      0.105     -2.243      0.025

 Variances
    F                  1.000      0.000    999.000    999.000


QUALITY OF NUMERICAL RESULTS

     Condition Number for the Information Matrix              0.118E+00
       (ratio of smallest to largest eigenvalue)


RESULTS IN PROBABILITY SCALE

                    Estimate

 X1
    Category 1         0.630
    Category 2         0.370
 X2
    Category 1         0.615
    Category 2         0.385
 X3
    Category 1         0.486
    Category 2         0.514
 X4
    Category 1         0.434
    Category 2         0.566
 X5
    Category 1         0.468
    Category 2         0.532
 X6
    Category 1         0.519
    Category 2         0.481
 X7
    Category 1         0.621
    Category 2         0.379
 X8
    Category 1         0.502
    Category 2         0.498
 X9
    Category 1         0.256
    Category 2         0.744
 X10
    Category 1         0.450
    Category 2         0.550


IRT PARAMETERIZATION

                                                    Two-Tailed
                    Estimate       S.E.  Est./S.E.    P-Value

 Item Discriminations

 F        BY
    X1                 0.912      0.158      5.779      0.000
    X2                 1.509      0.236      6.400      0.000
    X3                 0.868      0.153      5.689      0.000
    X4                 0.699      0.139      5.029      0.000
    X5                 0.401      0.120      3.346      0.001
    X6                 1.479      0.227      6.510      0.000
    X7                 1.281      0.200      6.415      0.000
    X8                 0.395      0.120      3.308      0.001
    X9                 1.249      0.215      5.823      0.000
    X10                0.866      0.152      5.702      0.000

 Means
    F                  0.000      0.000      0.000      1.000

 Item Difficulties
    X1                 0.684      0.148      4.605      0.000
    X2                 0.438      0.094      4.670      0.000
    X3                -0.078      0.120     -0.645      0.519
    X4                -0.423      0.159     -2.669      0.008
    X5                -0.332      0.249     -1.333      0.183
    X6                 0.071      0.085      0.836      0.403
    X7                 0.508      0.107      4.767      0.000
    X8                 0.020      0.235      0.086      0.931
    X9                -1.103      0.157     -7.008      0.000
    X10               -0.272      0.126     -2.162      0.031

 Variances
    F                  1.000      0.000      0.000      1.000


     Beginning Time:  17:20:43
        Ending Time:  17:20:43
       Elapsed Time:  00:00:00



MUTHEN & MUTHEN
3463 Stoner Ave.
Los Angeles, CA  90066

Tel: (310) 391-9971
Fax: (310) 391-8971
Web: www.StatModel.com
Support: Support@StatModel.com

Copyright (c) 1998-2021 Muthen & Muthen
